ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -62,10 +62,12 @@ import SwiftUI
 struct MSFListCellView: View {
     @ObservedObject var state: MSFListCellState
     @ObservedObject var tokens: MSFListCellTokens
+    var style: MSFListCellStyle
 
-    init(state: MSFListCellState, windowProvider: FluentUIWindowProvider?) {
+    init(state: MSFListCellState, style: MSFListCellStyle = .normal, windowProvider: FluentUIWindowProvider?) {
```

Comment:
Passing the MSFListTokens instead of the Style seems better than having a style that maps to it.